docs/articles/clustering.md

```diff
@@ -122,9 +125,10 @@ frontend zot
 backend zot-cluster
     mode http
     balance roundrobin
-    server zot1 127.0.0.1:8081 check
```

Comment:
https://www.haproxy.com/blog/path-based-routing-with-haproxy
^ use this example instead

# route to a backend based on path's prefix
  use_backend zot-instance1 if { path_beg /v2/repo1/ }
  use_backend zot-instance2 if { path_beg /v2/repo2/ }
    
  backend zot-instance1
  server zot-server1 127.0.0.1:8080 check maxconn 30  
  
  backend zot-instance2
  server zot-server2 127.0.0.1:8081 check maxconn 30

Reply:
zot config dedupe=false

Reply:
it is this manual and dynamic config (repos may come and go anytime), which we improve upon in the new scale-out

Reply:
In basic clustering article, zot and HAProxy configs are revised. Please verify.